src/Context/Propagation/TextMapPropagator.php

```diff
@@ -14,7 +15,9 @@ public static function composite(TextMapPropagatorInterface ...$propagators): Te
             case 0:
                 return NoopTextMapPropagator::getInstance();
             case 1:
-                return $propagators[0];
+                assert($propagators !== []);
```

Comment:
is this possible, given the `count()` above?

Reply:
No, added to make [psalm happy](https://github.com/open-telemetry/opentelemetry-php/runs/6907417904?check_suite_focus=true#step:11:18).

Reply:
ok. you could also `@psalm-suppress` which might be clearer for future devs...?

Reply:
Seems like I accidentially used `/*` instead of `/**` when I tried that yesterday - changed.